dvc/repo/destroy.py

```diff
@@ -15,4 +16,5 @@ def _destroy_stages(repo):
 def destroy(repo):
     _destroy_stages(repo)
     repo.close()
+    destroy_dvcignore(repo.root_dir)
```

Comment:
I'd just prefer keeping things simple here:
```suggestion
    remove(os.path.join(repo.root_dir, DvcIgnore.DVCIGNORE_FILE)
```

Reply:
I think it makes sense to add `destroy()` next do `init()` to make it easier to keep init/destroy in sync, although I worry it's some form of premature optimization